app/api/email_notifications.py

```diff
@@ -91,7 +93,8 @@ class EmailNotificationDetail(ResourceDetail):
     """
     Email notification detail by ID
     """
-    decorators = (is_user_itself,)
+    decorators = (api.has_permission('is_user_itself', fetch="user_id", fetch_as="id",
+                  model=EmailNotification, fetch_key_url="id"),)
```

Comment:
Not an issue but fetch_key_url is not required parameter and by default it is "id"